imap_processing/cdf/utils.py

```diff
@@ -26,8 +26,8 @@ def load_cdf(
 
     Parameters
     ----------
-    file_path : Path
-        The path to the CDF file.
+    file_path : Path | ImapFilePath
```

Comment:
This should be in the `load_cdf()` declaration, and numpy doc style uses the word "or" I believe.
```suggestion
    file_path : Path or ImapFilePath
```